Update Digital Credentials Hierarchy and Taxonomy (#945)
italia/eidas-it-wallet-docs: docs/en/registry.rst

Digital Credentials Hierarchy
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> Digital Credentials recognized within the IT-Wallet ecosystem are classified and standardized according to the following multi-level hierarchical model designed to improve semantic clarity, credential discovery, and compatibility with both credential-specific and claim-based verification workflows.

Reviewer: ```suggestion
Digital Credentials recognized within the IT-Wallet ecosystem are classified and standardized according to the following multi-level hierarchical model designed to improve semantic clarity, Credential discovery, and compatibility with both credential-specific and claim-based verification workflows.
```

   * - *EMPLOYMENT*
     - * Employment Documents
       * Employment Status
       - Digital Employment Contract
       - Curriculum Vitae (CV)
       - Residence Permit
       - Employment Status Certificate
       - INPS Contribution Record
     - Credentials that provide official statements, confirmations of status, or certifications issued by authorities.
   * - *MOBILITY AND TRAVEL*
     - * Licenses and Authorizations
       * Vehicle Documents
       * Subscriptions
       * Travel Documents 
       * Pet Travel Documents
       * Travel Insurance
       * Bookings
       * Discounts and Benefits
       - Mobile Driving License
       - Boating License
       - Vehicle Registration Certificate
       - Digital RCA Insurance
       - Vehicle Inspection Certificate
       - Green Card / International Insurance
       - Public Transport Pass
       - Telepass Subscription
       - Passport
       - European Pet Passport
       - Travel Tickets (air, train, etc.)
       - Travel Insurance Policy
       - Hotel Reservation
       - Discount Cards
       - Tourist Benefits

>      - Credentials that authorize, enable, or certify the right to move, drive, travel, or access transport services, including personal licenses, vehicle documents, travel         documents, and related insurance or entitlement proofs, as recognized by national or international regulations.

Reviewer: ```suggestion
     - Credentials that authorize, enable, or certify the right to move, drive, travel, or access transport services, including personal licenses, vehicle documents, travel documents, and related insurance or entitlement proofs, as recognized by national or international regulations.
```

   * - *BONUSES*
     - * Economic Benefits and Allowances
       * Incentives and Vouchers
       * Health and Wellbeing Bonuses
       - Family Allowance Credential
       - Unemployment Benefit Credential
       - Digital Voucher
       - Purchase Incentive Credential
       - Cashback Eligibility Credential
       - Healthcare Bonus Credential
       - Mental Health Support Voucher
       - Sports and Physical Activity Bonus

>      - Credentials that attest the right to access or receive economic benefits, subsidies, incentives, or vouchers, including eligibility criteria, allocation status, and          applicable usage constraints, as defined by relevant policies or regulations.

Reviewer: ```suggestion
     - Credentials that attest the right to access or receive economic benefits, subsidies, incentives, or vouchers, including eligibility criteria, allocation status, and applicable usage constraints, as defined by relevant policies or regulations.
```

Digital Credentials Hierarchy
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> Digital Credentials recognized within the IT-Wallet ecosystem are classified and standardized according to the following multi-level hierarchical model designed to improve semantic clarity, credential discovery, and compatibility with both credential-specific and claim-based verification workflows. 

Reviewer 2: ```suggestion
Digital Credentials recognized within the IT-Wallet ecosystem are classified and standardized according to the following multi-level hierarchical model designed to improve semantic clarity, Credential discovery, and compatibility with both credential-specific and claim-based verification workflows. 
```

Reviewer 3: ```suggestion
Digital Credentials recognized within the IT-Wallet ecosystem are classified and standardized according to the following multi-level hierarchical model designed to improve semantic clarity, Credential discovery, and compatibility with both Credential-specific and claim-based verification workflows.
```

Digital Credentials Hierarchy
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Digital Credentials recognized within the IT-Wallet ecosystem are classified and standardized according to the following multi-level hierarchical model designed to improve semantic clarity, credential discovery, and compatibility with both credential-specific and claim-based verification workflows. 

The hierarchy is defined as follows:

**Domain**

> A **Domain** represents a high-level thematic area grouping credential families that relate to the same broad context (e.g., Identity, Health, Education, Mobility).  

Reviewer 3: ```suggestion
A **Domain** represents a high-level thematic area grouping Credential families that relate to the same broad context (e.g., Identity, Health, Education, Mobility).  
```

Domains provide a top-level organizational layer.

**Class (Credential Family)**

> A **Class** represents a family of credentials sharing similar nature, function, or structure (e.g., Identity Documents, Civil Status Certificates).  

Reviewer 3: ```suggestion
A **Class** represents a family of Credentials sharing similar nature, function, or structure (e.g., Identity Documents, Civil Status Certificates).  
```

Each Class SHOULD define:

- a stable Class identifier (URI),  

> - the expected semantics of the credential family.

Reviewer 3: ```suggestion
- the expected semantics of the Credential Family.
```

> - the expected semantics of the credential family.

Reviewer 3: ```suggestion
- the expected semantics of the Credential Family.
```

> Classes enable Relying Parties and Wallets to request or match credentials based on their type category.

Reviewer 3: ```suggestion
Classes enable Relying Parties and Wallet Solutions to request or match Credentials based on their type category.
```

**Credential Type (Subclass)**

> A **Credential Type(Subclass)** represents a specific credential instance within a Class (e.g. Passport, Birth Certificate, Mobile Driving License).  

Reviewer 3: ```suggestion
A **Credential Type (Subclass)** represents a specific Credential within a Class (e.g. Passport, Birth Certificate, Mobile Driving License).  
```

Each Credential Type SHALL include:

- a unique identifier (e.g., VCT, schema URI),  

> - the issuing authority,  

Reviewer 3: @AntoPaparella Is it the Credential Issuer or the Authentic Source? Issuing authority seams not to be a defined term.
cc @peppelinux

> - the issuing authority,  

Reviewer: ```suggestion
- the Credential Issuer identifier,  
```

@RosaliaGaleano for certain it must contain the credential issuer unique identifier, therefore here my suggestion.

> - the set of attributes that MAY be included in presentations.

Reviewer 3: ```suggestion
- the set of Attributes that MAY be included in presentations.
```

Credential Types enable precise targeting for compliance-driven or regulation-mandated verification flows.

**Purpose (Verification Intent)**

A **Purpose (Verification Intent)** describes *why* a credential may be requested by a Relying Party (e.g., Identity Verification, Age Verification, Eligibility for specific services).  

> Purposes MUST NOT represent categories of credentials. Purposes SHALL describe **verification outcomes**, not credential categories.

Reviewer 3: ```suggestion
Purposes SHALL describe **verification outcomes**.
```

The **Taxonomy** provides the semantic foundation for Digital Credential interoperability by maintaining the authoritative vocabulary for organizing Credentials within the IT-Wallet ecosystem. The taxonomy is neutral with respect to the Credential format. 

> The taxonomy provides, in a single resource, the hierarchical classification system organizing Domains, Classes and Purposes that can be applied to credential types, supporting authorization policy evaluation and ecosystem-wide standardization.

Reviewer 3: ```suggestion
The Taxonomy provides, in a single resource, the hierarchical classification system organizing Domains, Classes and Purposes that can be applied to Credential Types, supporting authorization policy evaluation and ecosystem-wide standardization.
```

**Taxonomy Objectives:**

1. **Semantic Foundation**: Establish standardized vocabulary for domains and purposes across the ecosystem
2. **Policy Framework**: Enable structured authorization decisions based on hierarchical classification
3. **Interoperability**: Ensure consistent interpretation of credential classifications

> 4. **Extensibility**: Support evolution of the ecosystem with new domains, classes, credential types and purposes

Reviewer 3: ```suggestion
4. **Extensibility**: Support evolution of the ecosystem with new Domains, Classes, Credential Types and Purposes
```

5. **Cross-Border Compliance**: Align with EU regulatory requirements and international standards

**Taxonomy Structure:**

The taxonomy maintains a four level hierarchical structure:

- **Domains**: Top-level classification representing broad functional areas (e.g., IDENTITY, HEALTH, FINANCIAL)

> - **Class(Credential Family)**: Family of credentials sharing similar function, structure, or legal meaning (e.g., Identity Documents, Civil Status Certificates, Professional Licenses)

Reviewer 3: ```suggestion
- **Class(Credential Family)**: Family of Credentials sharing similar function, structure, or legal meaning (e.g., Identity Documents, Civil Status Certificates, Professional Licenses)
```

> - **Credential Type (Subclass)**: Specific credential definition issued by an authority (e.g., Passport, Birth Certificate, Mobile Driving License).

Reviewer 3: ```suggestion
- **Credential Type (Subclass)**: Specific Credential definition issued by an authority (e.g., Passport, Birth Certificate, Mobile Driving License).
```

> - **Purpose (Verification Intent)**: Verification objectives that a credential can satisfy (e.g., Identity Verification, Age Verification, Eligibility for specific services).

Reviewer 3: ```suggestion
- **Purpose (Verification Intent)**: Verification objectives that a Credential can satisfy (e.g., Identity Verification, Age Verification, Eligibility for specific services).
```

- **Claims Registry**: Individual claims catalog
- **AS Registry**: Authentic Sources declare capabilities using taxonomy classifications

> - **Digital Credentials Catalog**: Credential types specify domains, classes and purposes

Reviewer 3: ```suggestion
- **Digital Credentials Catalog**: Credential Types specify Domains, Classes and Purposes
```

- **Authorization Policies**: Policy evaluation leverages taxonomy structure for access control decisions

> The taxonomy is accessible through the dedicated taxonomy endpoint as defined in the registry discovery mechanism and is maintained by the Supervisory Body to ensure regulatory compliance and semantic consistency.

Reviewer 3: ```suggestion
The Taxonomy is accessible through the dedicated taxonomy endpoint as defined in the registry discovery mechanism and is maintained by the Supervisory Body to ensure regulatory compliance and semantic consistency.
```

.. code-block:: json

  {
    "domain.identity.name": "IDENTITY",
    "domain.class": "Identification Documents",
    "domain.subclass": "Passport",
    "domain.identity.description": "Credentials that establish or confirm a person’s legal identity and personal status",

>     "purpose.person_identification.name": "Identity verification", "Age verification", "Verification of citizenship or nationality"

Reviewer 4: @AntoPaparella @RosaliaGaleano should be "purpose.person_identification.name": "Identity verification", right?

>     "purpose.person_identification.name": "Identity verification", "Age verification", "Verification of citizenship or nationality"

Reviewer 3: That's ok @pcerqua-ipzs. The new sentence could be:
```suggestion
    "purpose.person_identification.name": "Identity verification"
```
